### PR TITLE
Added support for showing times from different timezones

### PR DESCRIPTION
**Pull request by juhamust**
- New day/night indicator (using font awesome icon definitions from Mozaik theme, if available)
- New optional config parameter: timezone where value can be any of the documented ones: http://momentjs.com/timezone/docs/ (example: America/Toronto)
- New optional config parameter: info (timezone, date, time or <text>)
- New dependencies: moment-timezone, react-classset
- Removed obsolete(?) dependencies

NOTES: 
- Not sure why some modules like `react` is in devDependencies? Assuming those are needed runtime, too? Or are those to be found from Mozaik dependencies?
- In the local environment I'm getting following errors from Mozaik/React (possibly due to npm link usage?)

```
ReactMount.findComponentRoot: 
  Uncaught TypeError: Cannot read property 'firstChild' of undefined
```

### CODE REVIEW

**Comment by plouc** [comment]
react is in devDependencies because it should not be installed when used as a submodule, the Error you got is due to the fact you have two react libraries loaded, the one from mozaik-ext-time plus the mozaik one.

File: README.md [hunk 1]
```diff
@@ -4,19 +4,33 @@
...
 
-> Not really useful, but every dashboard composition tool must have one :)
+Show analog clock from local or anoter timezone.
 
 ![clock](https://raw.githubusercontent.com/plouc/mozaik-ext-time/master/preview/time.clock.png)
 
 ### parameters
 
-*This widget has no parameter*
+Both parameters are optional:
```
Comment (plouc) [inline README.md]: You should use the same format used on other extensions for parameters (https://github.com/plouc/mozaik-ext-jenkins/blob/master/README.md#parameters)

File: src/components/Clock.jsx [hunk 2]
```diff
@@ -39,10 +44,32 @@ var Clock = React.createClass({
             transform: 'rotate(' + secondsScale(this.state.seconds) + 'deg)'
         };
 
+        // Day/night indicator
+        var isDay = true;
+        if (this.state.moment.hours() >= 18 || this.state.moment.hours() < 6) {
```
Comment (plouc) [inline src/components/Clock.jsx]: those values should be props, because it depends on where you live

File: README.md [hunk 3]
```diff
@@ -4,19 +4,33 @@
 
 ## Time — Clock
 
-> Not really useful, but every dashboard composition tool must have one :)
+Show analog clock from local or anoter timezone.
```
Comment (plouc) [inline README.md]: I used blockquote for every other extensions docs

File: package.json [hunk 4]
```diff
@@ -15,14 +15,17 @@
   },
   "homepage": "https://github.com/plouc/mozaik-ext-time",
   "dependencies": {
-    "d3": "^3.5.5"
+    "d3": "^3.5.5",
+    "react": "^0.12.2",
```
Comment (plouc) [inline package.json]: `react` Must be in devDependencies

File: src/components/Clock.jsx (hunk 2 above)
Comment (juhamust) [inline src/components/Clock.jsx]: You mean the the values 6 and 18 (approximate start and end of the daylight)? Yes, those values changes a lot: I live in up here in Finland where the amount of daylight changes constantly. So, fixed values can surely be parametrized but I don't think one keeps adjusting them throughout the year :) Fetching the _actual_ info from somewhere would be nice thouch, though.

Anyway, I'll add them as params if that's what you're after.

**Comment by juhamust** [comment]
Updated the PR. Please review.

**Comment by plouc** [comment]
I've published a new version of the _mozaik-ext-time_ package which includes your improvements, I've also added some basic tests to the repo.

Thanks for your contribution

**Pull request merged**